### PR TITLE
Feature: Form validation.

### PR DESCRIPTION
**Pull request by jlcarrascof**
In this branch, we worked pair to pair with my coding partners **@jicamargo** and @lerfast   and we did the following:

- [ ] Add **formvalidation.js** file.
- [ ] Add a **small tag** to show the error message dynamically.
- [ ] Add an **invalid-input class on style.css** to make a better UX.
- [ ] Add email validation logic on **formvalidation.js** file.

 **OBSERVATION:** 

**Because I was a bit behind of my coding partners this week, we did this job together but I just submitted because the previous one was very tough for me. Thanks for your comprehension.**

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has no visible contribution from your pair programming partner. Kindly check [Work and Submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md).
Screenshot 2023-02-08 at 9 56 02 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jlcarrascof** [comment]
Thanks @huseyinbiyik .... My coding partner this week @jicamargo just did an update. We worked together but he's in a personal activity today. Thanks for your comprehension.

Javier.-

**Review by alexander16108 — APPROVED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that has no visible contribution from your pair programming partner. Kindly check [Work and Submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by jlcarrascof** [comment]
Thanks @alexander16108 

I just modified the main comment in the pull request, doing this 

**OBSERVATION:**

**Because I was a bit behind of my coding partners this week, we did this job together but I just submitted because the previous one was very tough for me. Thanks for your comprehension.**

I hope that you or other buddy as code reviewer, can help me with this issue. I spent almost two days doing the previous one and in this that it was the easier project of the week, **looks like I will have some troubles to approve although everything is working fine and I'm giving you guys, a reasonable reason**.

Thanks in advance,

Javier.-

File: assets/css/styles.css [hunk 1]
```diff
@@ -42,9 +42,8 @@ a {
 
 #error-message {
   color: red;
-  font-size: 1.5rem;
+  font-size: 25px;
   margin-bottom: 1.5rem;
-  
 }
```
Comment (Owusu-Desmond) [inline assets/css/styles.css:47]: - [ ] Your email validation works fine :+1:  but from the project requirement, If the validation is not OK, you should show an error message to the user **near the submit button** informing them of the error, and the form is not sent. So make sure you display the error message near the submit button. As you can see below your error message is not near the submit button :smile: 

  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @jlcarrascof and @lerfast 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/styles.css (hunk 1 above)
Comment (jlcarrascof) [reply assets/css/styles.css:47]: Thanks, dear friend **@Owusu-Desmond** I'll fix it immediately. 

Javier.-

**Comment by jlcarrascof** [comment]
SCREENSHOT

Thanks brother @Owusu-Desmond 

I'm in a hurry because I spent so much time in the previous project; I put the message in an innappropriate place, but as you can see above, I fixed it.

Kind regards,

Javier.-

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @jlcarrascof ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jlcarrascof** [comment]
Thanks **@1GORDON** and **@Owusu-Desmond** for your valuable feedback.

Thanks also to my coding partners: **@jicamargo**  and **@lerfast** 

Javier.-

**Pull request merged**